src/utils/ValidateConfigs.ts

```diff
@@ -0,0 +1,48 @@
+import { Component, ReactElement, createElement } from "react";
+import { TagContainerProps } from "../components/TagContainer";
+import { Alert } from "../components/Alert";
+
+export class ValidateConfigs extends Component<TagContainerProps & { showOnError: boolean }, {}> {
+    render() {
+        const message = ValidateConfigs.validate(this.props);
+
+        if (message) {
+            const alertClassName = "widget-tag-text-alert";
+            if (this.props.showOnError) {
+                return createElement("div", { className: "widget-tag-text-invalid" },
+                    createElement(Alert, {
+                        bootstrapStyle: "danger",
+                        className: alertClassName,
+                        message
+                    }),
+                    this.props.children as ReactElement<TagContainerProps>);
+            } else {
+                return createElement(Alert, {
+                    bootstrapStyle: "danger",
+                    className: alertClassName,
+                    message
+                });
+            }
+        }
+
+        return this.props.children as ReactElement<TagContainerProps>;
+    }
+
+    static validate(props: TagContainerProps): string {
+        const widgetName = "Tag";
+
+        if (props.lazyLoad) {
```

Comment:
Combine the nested ifs eg  `if (props.lazyLoad && !props.enableSuggestions)` ...

Reply:
Done